package.json

```diff
@@ -22,7 +22,7 @@
     "test": "mocha",
     "lint-staged": "lint-staged",
     "coverage-report": "ts-node util/gen_coverage_report.ts",
-    "util": "ts-node util/index.ts",
```

Comment:
My other feedback here is that it's kinda unfortunate that you have to run this through `npm run util` too and any other way of running it is a mistake.  Other scripts can be run by running the script directly and this is the only weird one.

Is there some way that you can do this jsdom-global dance inside the script itself?

Reply:
It should be run-able directly with e.g. `npx ts-node -r jsdom-global/register ...`. The jsdom-global can be auto-require'd by ts-node or by node itself (if you're somehow working with a compiled `.js` version of the script).

As for doing the dance in the file itself, from everything I tested it's not possible due to the way tsc is compiling the script (essentially HTMLElement is getting referenced before `jsdom-global/register` can load, even if `jsdom-global/register` is the first import in the script).

The more glaring issue is with all of the files registered to the `util/index.ts` helper, basically none of them can be run directly, they all have to be run through `util/index.ts`, not just this script. I'm pretty sure there's a way to write them so that they can be (by checking process.args for if it's being run directly, and calling the appropriate function), but maybe open an issue for that so it doesn't get lost?